src/MongoDBQueueServiceProvider.php

```diff
@@ -51,6 +54,15 @@ protected function registerFailedJobServices()
      */
     protected function mongoFailedJobProvider(array $config): MongoFailedJobProvider
     {
-        return new MongoFailedJobProvider($this->app['db'], $config['database'], $config['table']);
+        if (! isset($config['collection']) && isset($config['table'])) {
```

Comment:
Should it be an outright exception to specify _both_ `table` and `collection`? As-is, I assume `table` would just be ignored.

Reply:
Yes, `table` will be ignored if `collection` is set. Allowing both make migration easier as you can specify both during transition.